composer.json

```diff
@@ -81,6 +81,11 @@
                 }
             }
         },
+        "patches": {
+            "drupal/entity_reference_revisions": {
+                "https://www.drupal.org/project/entity_reference_revisions/issues/2937835": "https://www.drupal.org/files/issues/2021-03-26/entity_reference_revisions-field_formatter_label-2937835-36.patch"
```

Comment:
add a line in the readme of the submodule mentioning that this patch is needed and whatever other lines from composer 
something like this https://github.com/openeuropa/oe_theme/tree/3.x/modules/oe_theme_content_project#required-contrib-modules
and for the patch something like this
https://github.com/openeuropa/oe_content/blob/2.x/modules/oe_content_project/README.md